Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM sbtscala/scala-sbt:eclipse-temurin-jammy-19.0.1_10_1.8.2_3.2.2
+
+WORKDIR /d4-chat
+ADD . /d4-chat
+
+EXPOSE ${APP_PORT}
+
+CMD ["sbt", "run"]
```

Comment:
It's not a good idea to run the app via sbt on a server due to the additional overhead and resource usage